### PR TITLE
special case for devenv.exe

### PR DESCRIPTION
**Pull request by Geobert**
Fix https://github.com/alexcrichton/cc-rs/issues/325

### CODE REVIEW

File: src/windows_registry.rs [hunk 1]
```diff
@@ -622,6 +626,27 @@ mod impl_ {
...
+
+    fn find_devenv_vs15(target: &str) -> Option<Tool> {
+        let key = r"SOFTWARE\WOW6432Node\Microsoft\VisualStudio\SxS\VS7";
+        LOCAL_MACHINE
+            .open(key.as_ref())
+            .ok()
+            .and_then(|key| key.query_str("15.0").ok())
+            .map(|path| {
+                let path = PathBuf::from(path).join(r"Common7\IDE\devenv.exe");
+                println!("*** path: {:?}", path);
```
Comment (alexcrichton) [inline src/windows_registry.rs]: stray println?

**Comment by alexcrichton** [comment]
Thanks!

File: src/windows_registry.rs (hunk 1 above)
Comment (Geobert) [reply src/windows_registry.rs]: oops, fixed ^^

**Comment by Geobert** [comment]
Note that it's only for VS2017, if we need other version as well, I need to figure out their path and code the missing parts :)

**Pull request merged**